### DISCUSSION TITLE
Hitsounds sound weird after the update from 08/08

### DISCUSSION THREAD

**Discussion by TheDragula** [ppy/osu#14182] [Q&A] [answered]
After the update from 08/08 hitsounds sound like they are more in the background.
They are quieter and have like an effect on them.
The user Loreos even made a video about that comparing hitsounds from osu stable to the current osu lazer version (08/08)
https://osu.ppy.sh/comments/1773615
and the video
https://youtu.be/ZXNUAbYf790

If that is intentional, i would like to ask if we get an option to disable that.

**Comment by peppy** [comment]
I also felt like there was something different, especially on slider slider sounds. Will require further investigation. It may be related to looping logic or otherwise.

@TheDragula can you confirm that it still happens after today's hotfix release?

**Reply by TheDragula** [reply]
Yes, it's still the same.

**Comment by smoogipoo** [comment, marked as answer]
Thanks for the report. It'll be resolved in the next release.
